python/pyspark/sql/connect/plan.py

```diff
@@ -265,13 +277,13 @@ class Project(LogicalPlan):
 
     def __init__(self, child: Optional["LogicalPlan"], *columns: "ColumnOrName") -> None:
         super().__init__(child)
-        self._raw_columns = list(columns)
```

Comment:
Why this change?

Reply:
Because this one was only the exception that does not follow the variable naming style in this file. All are either `name` or `_name`, and the automatic printing logic relies on this naming.